src/test/base.sol

```diff
@@ -141,4 +141,36 @@ contract TestBase is DSTest {
         assertTrue(false);
     }
 
+    function assertEqApprox(uint256 _a, uint256 _b, uint256 _tolerance) internal {
+        uint256 a = _a;
+        uint256 b = _b;
+        if (a < b) {
+            uint256 tmp = a;
+            a = b;
+            b = tmp;
+        }
+        if (a - b > _tolerance) {
+            emit log_bytes32("Error: Wrong `uint' value");
+            emit log_named_uint("  Expected", _b);
+            emit log_named_uint("    Actual", _a);
+            fail();
+        }
+    }
+
+    function assertEqApproxBPS(uint256 _a, uint256 _b, uint256 _tolerance_bps) internal {
+        uint256 a = _a;
+        uint256 b = _b;
+        if (a < b) {
+            uint256 tmp = a;
+            a = b;
+            b = tmp;
+        }
+        if (a - b > mul(_b, _tolerance_bps) / 10 ** 5) {
```

Comment:
For basis points you should use 10^4, not 10^5: https://en.wikipedia.org/wiki/Basis_point

Reply:
Also since you've sorted so that `a > b`, I'd use `b` on the RHS of the `>` instead of `_b` so that you're always using the tighter of the two possible error bounds.

Reply:
Nvm the comment I just deleted if you saw it, since `_b` is interpreted as the expectation it's appropriate to use it as the basis for calculating the error margin.